src/services/completions.ts

```diff
@@ -2057,24 +2031,10 @@ namespace ts.Completions {
                 }
             }
 
-            const result: Symbol[] = [];
-            addPropertySymbols(baseSymbols, ModifierFlags.Private);
-            addPropertySymbols(implementingTypeSymbols, ModifierFlags.NonPublicAccessibilityModifier);
-            return result;
-
-            function addPropertySymbols(properties: ReadonlyArray<Symbol>, inValidModifierFlags: ModifierFlags) {
-                for (const property of properties) {
-                    if (isValidProperty(property, inValidModifierFlags)) {
-                        result.push(property);
-                    }
-                }
-            }
-
-            function isValidProperty(propertySymbol: Symbol, inValidModifierFlags: ModifierFlags) {
-                return !existingMemberNames.get(propertySymbol.escapedName) &&
-                    propertySymbol.getDeclarations() &&
-                    !(getDeclarationModifierFlagsFromSymbol(propertySymbol) & inValidModifierFlags);
-            }
+            return baseSymbols.filter(propertySymbol =>
+                !existingMemberNames.has(propertySymbol.escapedName) &&
+                    !!propertySymbol.declarations &&
+                    !(getDeclarationModifierFlagsFromSymbol(propertySymbol) & ModifierFlags.Private));

```

Comment:
The change to use `Private` all the time instead of `NonPublicAccessibilityMOdifier` is technically a change in behavior in the following situation (simplification of `completionEntryForClassMembers2.ts`):

```ts
class C {
    protected m(): void {}
}
interface I extends C {}
class D implements I {
    /**/
}
```

Previously we would not provide a completion for `m`, now we will. But I don't think this matters because the situation is impossible to implement -- you can't override `m` because it's private, but you have to implement it to implement the interface, so it's just illegal to implement `C` without extending from it anyway.